src/_includes/layouts/membership.liquid

```diff
@@ -1,3 +1,30 @@
 ---
 layout: layouts/base.liquid
 ---
+<div class="bg-parallax-photo">
+  <div
+    :speed-factor="0.8"
+    sectionClass="parallax-outside"
+    containerClass="parallax-inside">
+    <img src="/assets/img/bg-triangle-pink3.png" class="float-right" />
+  </div>
+</div>
+<div class="container py-5">
+  <div class="row justify-content-md-between">
+    <div class="col-md-6 py-md-5">
+      <h1 class="text-magenta my-md-5">{{ membershipHeader }}</h1>
+      {% if membershipAnnouncement != "" %}
```

Comment:
Is this saying if the membership announcement is not blank, then display the membership announcement?

Reply:
Oh I see what you are saying. How can I distinguish between users who are members and those who are not?

Reply:
You shouldn't have to because there aren't different users for the website. That is more of a public announcement. Remember, there isn't a database, just static files being served. So there isn't a member user.

Reply:
I had originally included "hiding the announcement box if there is no content" as a to-do item for this task, so that's why Bella included it! However since you highlighted the announcement box is used year-round, rather than temporarily, Bella can remove these two lines and assume the announcement box content is always populated.

Reply:
Resolved in [5d1c5db](https://github.com/kindsys/digital-services-coalition/pull/23/commits/5d1c5dbef73838405a581cead1eecd7c5f0e07ee)